packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewClientFlutterApiImpl.java

```diff
@@ -71,26 +73,31 @@ static GeneratedAndroidWebView.WebResourceRequestData createWebResourceRequestDa
   public WebViewClientFlutterApiImpl(
       BinaryMessenger binaryMessenger, InstanceManager instanceManager) {
     super(binaryMessenger);
+    this.binaryMessenger = binaryMessenger;
     this.instanceManager = instanceManager;
   }
 
   /** Passes arguments from {@link WebViewClient#onPageStarted} to Dart. */
   public void onPageStarted(
       WebViewClient webViewClient, WebView webView, String urlArg, Reply<Void> callback) {
-    final Long webViewIdentifier = instanceManager.getIdentifierForStrongReference(webView);
-    if (webViewIdentifier == null) {
-      throw new IllegalStateException("Could not find identifier for WebView.");
-    }
+    final WebViewFlutterApiImpl webViewFlutterApi =
```

Comment:
Optional: Should we just make a private field with this API on object construction, so we don't need the boilerplate of creating one in every single callback? I know it's cheap to create, but it should also be very cheap to just keep it around.